ironfish-cli/Dockerfile

```diff
@@ -1,13 +1,17 @@
 FROM node:16.15.1-bullseye as build
 ENV PATH="/root/.cargo/bin:${PATH}"
 
-COPY ./ ./
-
 RUN \
     apt-get update && \
-    apt-get install jq rsync gcc cmake python -y && \
-    curl https://sh.rustup.rs -sSf | sh -s -- -y && \
-    ./ironfish-cli/scripts/build.sh
+    apt-get install jq rsync gcc cmake python -y
```

Comment:
Splitting these into individual run steps lets us use docker layer caching so we don't have to re-install these and rustup every time

Reply:
Yeah. We can do this because this step in the multi stage build is dropped. If this was in the final image, docker recommends to reduce layers by combining them to avoid packages writing to the same files in older layers. https://docs.docker.com/develop/dev-best-practices/#how-to-keep-your-images-small